assets/js/hooks/cell.js

```diff
@@ -215,6 +211,16 @@ const Cell = {
       }, 0);
     });
 
+    liveEditor.onSelectionChange((update) => {
```

Comment:
Nit: I was thinking let's move `!update.state.selection.eq(update.startState.selection)` check to the live editor and we can emit only the new selection (though we don't really need to use it here :p).